app/views/group_members/index.html.erb

```diff
@@ -23,7 +23,7 @@
     end %>
 
     <%= table.with_body do |body| %>
-      <% @group.memberships.each do |membership| %>
+      <% @group.memberships.joins(:user).order("users.name").each do |membership| %>
```

Comment:
I wonder whether the ordering should be in the model, maybe as a method - rather than being in the view?

Reply:
Yeah, my initial thought was that it would be nice to have the ordering on the model, but to be honest the main reason I didn't do that is because I didn't know what to call it 😅 

Actually my first thought was that we could just put a scope on the `users` association in the `Group` model, so it's always sorted, but then I realised that we do actually need to iterate by memberships to get the user's role in the group. I then thought about putting the scope on the memberships, but that feels like it would be inefficient.

Anyway it would be fairly straightforward to add an [association extension](https://guides.rubyonrails.org/association_basics.html#association-extensions) that does this little snippet, similar to what we do in https://github.com/alphagov/forms-admin/commit/877e26b1be4cea82b30e48e52b18736c847e9c64, but yeah I don't know what to call it 😅 Any suggestions?

Reply:
`ordered_memberships`?

Reply:
@stephencdaly ooh interesting idea, I hadn't considered that as an option, I was sort of thinking we needed to shoehorn "by user name" in there somehow.

What about `memberships.ordered` (having an association extension instead of a plain method)?

Reply:
Oh, I didn't properly look at what the association extensions did. Yeh that would make sense to me!

Reply:
Cool I've made that change now!